examples/cpp_classification/readme.md

```diff
@@ -0,0 +1,75 @@
+---
+title: CaffeNet C++ Classification example
+description: A simple example performing image classification using the low-level C++ API.
+category: example
+include_in_docs: true
+priority: 10
+---
+
+# Classifying ImageNet: using the C++ API
+
+Caffe, at its core, is written in C++. It is possible to use
+the C++ API of Caffe to implement an image classification application
+similar to the Python code presented in one of the Notebook example.
+
```

Comment:
Thanks for the example. As a finishing touch, could you include a reference to the `caffe` [command line tool source](https://github.com/BVLC/caffe/blob/master/tools/caffe.cpp) as a further example of C++ code? Although I might hope this is an obvious place to check, it seems it has been overlooked in the past.

Adding a pointer after this intro paragraph could help as a stepping stone to seeing more C++ code.